libraries/AP_HAL_Linux/SPIDevice.cpp

```diff
@@ -125,7 +125,7 @@ SPIDesc SPIDeviceManager::_device[] = {
 };
 #elif CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_AERO
 SPIDesc SPIDeviceManager::_device[] = {
-    SPIDesc("aeroio", 1, 1, SPI_MODE_0, 8, SPI_CS_KERNEL,  10*MHZ, 10*MHZ),
+    SPIDesc("aeroio", 1, 1, SPI_MODE_0, 8, SPI_CS_KERNEL,  4*MHZ, 4*MHZ),
```

Comment:
It should handle 10MHz... at least with the RCOutput implementation I tested a long time ago

Reply:
ok, then why is the intel tool spi_xfer limited to 4MHz? I can test it at 10MHz though.
Thanks

Reply:
Maybe a worse spi implementation on that aero_sample... Or simply by no reason. I'm not sure I still have the old one at 10MHz

Reply:
I will test it at 10MHz, if it works I'll leave that spidesc at 10*MHz

Reply:
what is this rctap output you use?

Reply:
The motor/ESC that come on the RTF kit. It uses uart for communication. If you use CAN for motors it would be better not to use TAP so you free up the uart for other use.